cloud_chemistry/prototype/config/reactions.json

```
@@ -0,0 +1,13 @@
+{
+  "version": "1.0.0",
+  "name": "Hard-coded Cloud Chemistry",
+  "reactions": [
+    {
+      "type": "HL_PHASE_TRANSFER",
+      "gas phase": "gas",
+      "gas-phase species": "FOO",
+      "aerosol phase": "aqueous aerosol",
```

```suggestion
      "aerosol phase": "aqueous",
```
Just for consistency with `phases.json`

---

Agree, done!